app/code/Magento/Quote/Api/Data/ShippingAssignmentInterface.php

```diff
@@ -34,6 +34,18 @@ public function getItems();
      */
     public function setItems($value);
 
+
```

Comment:
We had a discussion regarding implementation of the Visitor pattern to integrate MSI with Quote. Maybe it is better to research that solution?